hs_core/templates/pages/genericresource.html

```diff
@@ -1000,25 +1000,25 @@ <h3>Resource Specific</h3>
       const STATIC_URL = "{{ STATIC_URL }}";
       const USERS_JSON = {{ users_json|safe }};
       const LAST_CHANGED_BY = {{ last_changed_by|safe }};
-      const HAS_ACTIVE_CREATORS = "{{ has_active_creators }}" === "True";
       const AUTHORS = [
           {% for author in creators %}
-                {
-                    "id": "{{ author.id }}",
-                    "name": "{{ author.name|default:""|escapejs }}",
-                    "email": "{{ author.email|default:"" }}",
-                    "organization": "{{ author.organization|default:""|escapejs }}",
-                    "identifiers": {
-                        {% for name, link in author.identifiers.items %}
-                            "{{ name }}": "{{ link }}",
-                        {% endfor %}
-                    },
-                    "address": "{{ author.address|default:""|escapejs }}",
-                    "phone": "{{ author.phone|default:"" }}",
-                    "homepage": "{{ author.homepage|default:"" }}",
-                    "profileUrl": "{{ author.relative_uri|default:'' }}",
-                    "order": "{{ author.order }}"
+            {
+                "id": "{{ author.id }}",
+                "name": "{{ author.name|default:""|escapejs }}",
+                "email": "{{ author.email|default:"" }}",
+                "organization": "{{ author.organization|default:""|escapejs }}",
+                "identifiers": {
+                    {% for name, link in author.identifiers.items %}
+                        "{{ name }}": "{{ link }}",
+                    {% endfor %}
                 },
+                "address": "{{ author.address|default:""|escapejs }}",
+                "phone": "{{ author.phone|default:"" }}",
+                "homepage": "{{ author.homepage|default:"" }}",
+                "profileUrl": "{{ author.relative_uri|default:'' }}",
+                "order": "{{ author.order }}",
+                "is_active": "{{ author.is_active|default:"" }}",
```

Comment:
important addition in this PR: adding `is_active` into our JS object so it can be used on the frontend